app/Domains/Accounts/Actions/CreateAccount.php

```diff
@@ -32,7 +32,11 @@ public function handle(array $params): Account
     public function rules(): array
     {
         return [
-
+            'id' => ['required', 'uuid'],
```

Comment:
ID is autogenerated, it should not exist in the validation rule for create

Reply:
Removed ID